abi/abi.go

```diff
@@ -197,8 +241,12 @@ func (it Item) Uint8() uint8 {
 	return uint8(bint.Decode(it.d))
 }
 
-func (it Item) Uint64() uint64 {
```

Comment:
moved this around on purpose for easier readability (keeps things in a consistent order)